ios/ci_scripts/ci_post_clone.sh

```
@@ -7,7 +7,7 @@ set -e
 cd $CI_PRIMARY_REPOSITORY_PATH # change working directory to the root of your cloned repo.
 
 # Install Flutter using git.
-git clone https://github.com/flutter/flutter.git --branch 3.44.4 --depth 1 $HOME/flutter
+git clone https://github.com/flutter/flutter.git --branch 3.47.1 --depth 1 $HOME/flutter
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)